data/djangocon-2014/videos/a-nice-problem-to-have-django-under-heavy-load.json

```diff
@@ -1,23 +1,25 @@
 {
-  "category": "DjangoCon 2014",
-  "slug": "a-nice-problem-to-have-django-under-heavy-load",
-  "title": "A Nice Problem to Have: Django Under Heavy Load",
-  "summary": "By, Joshua \"jag\" Ginsberg\n\"Don't prematurely optimize. Get your project to v1.0.\" This is a mantra often repeated in the Djangoverse. But what happens after v1.0 launch when your awesome site is being crushed by traffic? Scaling Django under load means finding bottlenecks, leveraging new tools, and customizing code. This talk will show you how it's done.\n\nHelp us caption & translate this video!\n\nhttp://amara.org/v/FOPX/",
-  "description": "",
-  "quality_notes": "",
-  "language": "English",
-  "copyright_text": "",
-  "thumbnail_url": "https://i.ytimg.com/vi/gukzap2BU2M/maxresdefault.jpg",
-  "duration": 2547,
+  "category": "DjangoCon 2014", 
+  "slug": "a-nice-problem-to-have-django-under-heavy-load", 
+  "title": "A Nice Problem to Have: Django Under Heavy Load", 
+  "summary": "\"Don't prematurely optimize. Get your project to v1.0.\" This is a mantra often repeated in the Djangoverse. But what happens after v1.0 launch when your awesome site is being crushed by traffic? Scaling Django under load means finding bottlenecks, leveraging new tools, and customizing code. This talk will show you how it's done.\n\nHelp us caption & translate this video!\n\nhttp://amara.org/v/FOPX/", 
+  "description": "", 
+  "quality_notes": "", 
+  "language": "English", 
+  "copyright_text": "", 
+  "thumbnail_url": "https://i.ytimg.com/vi/gukzap2BU2M/maxresdefault.jpg", 
+  "duration": 2547, 
   "videos": [
     {
-      "type": "youtube",
-      "length": 0,
-      "url": "https://www.youtube.com/watch?v=gukzap2BU2M"
+      "url": "https://www.youtube.com/watch?v=gukzap2BU2M", 
+      "length": 0, 
+      "type": "youtube"
```

Comment:
I'm kind of surprised these are changing order and that this PR seems to add whitespace to the end of lines. That's a bug in save/load_json functions, I think. They should round-trip files exactly.